extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -215,6 +216,10 @@ define('ext.wikia.adEngine.adContext', [
 		context.opts.disableSra = geo.isProperGeo(instantGlobals.wgAdDriverDisableSraCountries);
 		context.opts.isBLBMegaEnabled = geo.isProperGeo(instantGlobals.wgAdDriverBottomLeaderBoardMegaCountries);
 
+		context.opts.labradorTestGroup =
```

Comment:
Where is it used?

Reply:
It isn't, it is defined only for testing purposes